app/helpers/application_helper.rb

```diff
@@ -1,10 +1,6 @@
 module ApplicationHelper
   def home_link
-    link_url = "#"
-    link_url = providers_legal_aid_applications_url if request.path_info.include?("providers")
-    link_to_accessible(t("layouts.application.header.title"),
-                       link_url,
-                       class: "govuk-heading-m govuk-!-margin-bottom-0 govuk-header__link govuk-header__service-name")
+    request.path_info.include?("providers") ? providers_legal_aid_applications_url : "#"
```

Comment:
Concern: Is this correct, there is no link if a `citizen`?

Reply:
@naseberry I believe that is the current functionality for admins and citizens

Reply:
Is there a reason we're checking the path rather than something like `provider_signed_in?`?

Also, is there a reason we're passing the entire url rather than using the `_path` helper?